### PR TITLE
SageMaker on Flyte: installing SageMaker plugins [Alpha] 

### PR DESCRIPTION
**Pull request by bnsblue**
# TL;DR
This PR enables flyteplugins v0.4.0 for SageMaker support on flytepropeller and add config section needed by SageMaker to the sample config file

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [x] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
This PR enables flyteplugins v0.4.0 for SageMaker support and add config section needed by SageMaker to the sample config file

## Tracking Issue
https://github.com/lyft/flyte/issues/255

## Follow-up issue
https://github.com/lyft/flyte/issues/431

### CODE REVIEW

**Comment by bnsblue** [comment]
💨
